x-pack/plugins/streams_app/public/components/stream_detail_overview/index.tsx

```diff
@@ -8,15 +8,19 @@ import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
 import { calculateAuto } from '@kbn/calculate-auto';
 import { i18n } from '@kbn/i18n';
 import { useDateRange } from '@kbn/observability-utils-browser/hooks/use_date_range';
-import { StreamDefinition } from '@kbn/streams-plugin/common';
 import moment from 'moment';
 import React, { useMemo } from 'react';
+import type { APIReturnType } from '@kbn/streams-plugin/public/api';
 import { useKibana } from '../../hooks/use_kibana';
 import { useStreamsAppFetch } from '../../hooks/use_streams_app_fetch';
 import { ControlledEsqlChart } from '../esql_chart/controlled_esql_chart';
 import { StreamsAppSearchBar } from '../streams_app_search_bar';
 
-export function StreamDetailOverview({ definition }: { definition?: StreamDefinition }) {
+export function StreamDetailOverview({
+  definition,
+}: {
+  definition?: APIReturnType<'GET /api/streams/{id}'>;
```

Comment:
I _think_ `ReadStreamDefinition` is this type already?

Reply:
Ah, good catch, I thought I'd changed them all. I was using this before your PR added `ReadStreamDefinition`.